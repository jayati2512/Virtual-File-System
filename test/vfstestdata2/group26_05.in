createvfs vfs1 10
mountvfs vfs1
makedir /home/ dir1
makedir /home/ dir2
makedir /home/dir1/ dir3
makedir /home/dir2/ dir4
addfile /home/dir1/ apple /usr/include/ustat.h
addfile /home/ appy vfs2
addfile /home/ orange vfs3
addfile /home/ apricot vfs1
listfile /home/appy vfs1
updatefile /home/appy sample3
listfile /home/appy out2
searchfile o searchout.txt
removefile /home/dir1/apple
unmountvfs vfs1
