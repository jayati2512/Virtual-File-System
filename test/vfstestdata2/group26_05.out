createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
addfile_SUCCESS
addfile_FAILURE
addfile_FAILURE
addfile_FAILURE
listfile_SUCCESS
updatefile_FAILURE
listfile_SUCCESS
searchfile_SUCCESS
removefile_SUCCESS
unmountvfs_SUCCESS
